cfme/web_ui/toolbar.py

```diff
@@ -118,9 +118,13 @@ def pf_select(root, sub=None, invokes_alert=False):
                     sel.execute_script(
                         "return $('button:contains({})').trigger('click')".format(q_root))
                 except sel.NoSuchElementException:
-                    # The view selection buttons?
-                    sel.click("//li/a[@title={}]/*[self::i or self::img]/../..".format(q_root))
-
+                    try:
+                        sel.element("//a[@title = {}]".format(q_root))
+                        sel.execute_script(
+                            "return $('a[title={}]').trigger('click')".format(q_root))
+                    except sel.NoSuchElementException:
+                        # The view selection buttons?
+                        sel.click("//li/a[@title={}]/*[self::i or self::img]/../..".format(q_root))
```

Comment:
I think this has gotten too unwieldy now. Wondering if we can just run a single sel.elements with an OR in the XPATH?